osi_modelinternal.proto

```diff
@@ -22,7 +22,7 @@ message ModelInternal
     //
     // Low Level data should be transferred to receiving components by means of a separate data stream. Therefore they
     // are not part the top level SensorData message, as they are used only internally by some models.
-    optional LowLevelData low_level_data = 1;
+    optional FeatureData feature_data = 1;
```

Comment:
If we want to at least try to stay backward-compatible to a limited degree (otherwise we need to bump the version number to 3.0.0, I think, which might be needed due to the lane changes in any case), then the field ID of the new feature data should be 2, and the old 1 field should be retained as reserved, e.g.

    reserved 1;
    optional FeatureData feature_data = 2;